core/arch/opnd_shared.c

```diff
@@ -2082,9 +2082,9 @@ reg_get_bits(reg_id_t reg)
     if (reg >= DR_REG_START_XMM && reg <= DR_REG_STOP_XMM)
         return (byte)((reg - DR_REG_START_XMM) % 8);
     if (reg >= DR_REG_START_YMM && reg <= DR_REG_STOP_YMM)
-        return (byte)((reg - DR_REG_START_YMM) % 8);
+        return (byte)((reg - REG_START_YMM) % 8);
```

Comment:
See above, the REG_ enum/defines are deprecated. Use DR_REG_ version, please revert.